ebml_matroska.xml

```diff
@@ -27,17 +27,15 @@
     <extension type="webmproject.org" webm="1"/>
   </element>
   <element name="SegmentUUID" path="\Segment\Info\SegmentUUID" id="0x73A4" type="binary" range="not 0" length="16" maxOccurs="1">
-    <documentation lang="en" purpose="definition">A randomly generated unique ID to identify the Segment amongst many others (128 bits).
-It is effectively a Universally Unique IDentifier stored in binary form [@!RFC4122].</documentation>
+    <documentation lang="en" purpose="definition">A randomly generated unique ID to identify the Segment amongst many others (128 bits).</documentation>
```

Comment:
make it UUID version 4.

Reply:
I kept the default to all random but allowed UUID v4 where some bits a not random too.